tests/RuntimeExceptionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+* This file is part of Hashids.
+*
+* (c) Ivan Akimov <ivan@barreleye.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Hashids\Tests;
+
+use Hashids\Math;
+
+/**
+ * This is the runtime exception test class.
+ *
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+class RuntimeExceptionTest extends AbstractTestCase
+{
+    /**
+     * @expectedException \RuntimeException
+     */
+    public function testAdd()
+    {
+        Math::add(1, 2);
+    }
+
+    /**
+     * @expectedException \RuntimeException
+     */
+    public function testMultiply()
+    {
+        Math::multiply(1, 2);
+    }
+
+    /**
+     * @expectedException \RuntimeException
+     */
+    public function testDivide()
+    {
+        Math::divide(1, 2);
+    }
+
+    /**
+     * @expectedException \RuntimeException
+     */
+    public function testGreaterThan()
+    {
+        Math::greaterThan('1', '2');
+    }
+
+    /**
+     * @expectedException \RuntimeException
```

Comment:
PHPUnit is moving away for annotations like this. We should consider writing `$this->expectException(\RuntimeException::class);`

Reply:
Didn't know about that. Thanks!